nvme-print.c

```diff
@@ -267,6 +267,18 @@ static void nvme_show_persistent_event_entry_ehai(__u8 ehai)
 		"Event not associated with any port and PELPID does not apply");
 }
 
+static void add_bitmap(int i, __u8 seb)
+{
+        for (int bit = 0; bit < 8; bit++) {
+                if (nvme_pel_event_to_string(bit + i * 8)) {
+			if (nvme_pel_event_to_string(bit + i * 8))
```

Comment:
Duplicated the `if` condition checking line.

Reply:
I see, though this is not code I am adding, just moving around. Let's fix this up on top of this series. I'd like to avoid changing too many things in one go.